.pre-commit-config.yaml

```diff
@@ -76,7 +76,7 @@ repos:
           )$
 
   - repo: https://github.com/PyCQA/docformatter
-    rev: v1.7.5
+    rev: 06907d0
```

Comment:
what's uit for? why relying on a commit but not a stable tag?

Reply:
Documentation format tools for CI, it's a temporary solution: https://github.com/PyCQA/docformatter/issues/289